assets/styles/_base.scss

```diff
@@ -196,55 +222,75 @@ ol {
 
   li {
     margin-bottom: 0.5rem;
-    padding-left: 1.5rem;
+    padding-left: 1.7rem;
     position: relative;
 
-    &::before {
-      background-size: contain;
-      content: '';
+    .feather:first-child {
       display: block;
-      height: 1rem;
+      height: 20px;
       left: 0;
       position: absolute;
-      top: 0.25rem;
-      width: 1rem;
-    }
-  }
+      top: 2px;
+      width: 20px;
 
-  &--checkCircle {
-    li::before {
-      background-image: url('../assets/icons/check-circle.svg');
+      @media (min-width: $breakpoint-mobile-horizontal) {
+        top: 5px;
+      }
     }
   }
 }
 
 /* Tables */
 
 table {
-  border-collapse: collapse;
+  border: 1px solid var(--theme-color-text);
   border-spacing: 0;
   max-width: 100%;
+  width: 100%;
+}
+
+.boxRadius > thead {
+  th {
+    border: 1px solid  var(--theme-color-text); // fix IE border artifacts
+
+    &:first-child {
+      border-top-left-radius: 6px;
+    }
+
+    &:last-child {
+      border-top-right-radius: 6px;
+    }
+  }
 }
```

Comment:
Please test again if this is needed, since adding `overflow: hidden` to `.boxRadius`.

Reply:
yes, this is still needed for IE 😕